src/Core/AdminConsole/Enums/EventType.cs

```diff
@@ -58,6 +58,8 @@ public enum EventType : int
     OrganizationUser_Restored = 1512,
     OrganizationUser_ApprovedAuthRequest = 1513,
     OrganizationUser_RejectedAuthRequest = 1514,
+    OrganizationUser_Deleted = 1515, // Both user and organization user data were deleted
+    OrganizationUser_Left = 1516,    // User voluntarily left the organization
```

Comment:
Is this getting wired up in a separate PR?

Reply:
Yes, I'll do that here https://bitwarden.atlassian.net/browse/PM-10338